.pipelines/onebranch/templates/template-buildrp-buildaro.yml

```diff
@@ -0,0 +1,21 @@
+steps:
+- task: GoTool@0
+  displayName: Use Go 1.14
+  inputs:
+    version: '1.14'
+    GOPATH: $(Agent.TempDirectory)
+- task: Bash@3
+  displayName: ⚙️ Make ARO
+  inputs:
+    targetType: inline
+    script: |
+      mkdir -p $(Agent.TempDirectory)/src/github.com/Azure/
+      cp -rd $(Build.SourcesDirectory) $(Agent.TempDirectory)/src/github.com/Azure/ARO-RP
+      cd $(Agent.TempDirectory)/src/github.com/Azure/ARO-RP
+      ls
+      git rev-parse --short HEAD
+      git status --porcelain
```

Comment:
remove?

Reply:
Thanks - left over from previous debugging efforts to determine why binaries were being tagged "-dirty".  These have been removed.